packages/apiserver/src/resources/persons/person.schema.graphql

```diff
@@ -13,6 +13,7 @@ type Person {
   location: [Location!]
   messages: [Message]
   states: [BlockedUsers]
+  materials: [Material]
```

Comment:
do we need this filed in our `gql` schema?

Reply:
Not needed, good catch

Reply:
Deleted